sdk/servicebus/service-bus/test/perf/track-2/sendBatch.spec.ts

```diff
@@ -33,14 +33,13 @@ export class BatchSendTest extends ServiceBusTest<SendTestOptions> {
   constructor() {
     super();
     this.sender = this.sbClient.createSender(BatchSendTest.queueName);
-    this.sbMessage = {
+    const sbMessage = {
       body: Buffer.alloc(this.parsedOptions.messageBodySize.value!)
     };
+    this.batch = new Array(this.parsedOptions.numberOfMessages.value!).fill(sbMessage);
   }
 
   async runAsync(): Promise<void> {
-    await this.sender.sendMessages(
-      new Array(this.parsedOptions.numberOfMessages.value!).fill(this.sbMessage)
-    );
+    await this.sender.sendMessages(this.batch);
```

Comment:
Future concern: should we be using .tryAdd() ourselves or are we guaranteed that we'll never exceed the size of a single batch?

Reply:
Hmmm, makes sense to use tryAdd. I'll put a follow-up PR.

Reply:
#13489